### PR TITLE
docs: Represent the new changes in the charmed-magma repository

### PR DESCRIPTION
**Pull request by gruyaume**
## Summary

Improves documentation to represent the new changes in the charmed-magma repository:
- The deployment uses a new interface to receive TLS certificates from.

### CODE REVIEW

File: docs/readmes/orc8r/deploy_using_juju.md [hunk 1]
```diff
@@ -49,12 +49,11 @@ applications:
...
     options:
-      certificate: <your base64 encoded certificate>
-      private-key: <your base64 encoded private key>
-      ca-certificate: <your base64 encoded ca certificate>
+      generate-self-signed-certificates: true
+      ca-common-name: rootca.<your domain name>
 ```
 
-Replace the values in angle brackets.
+> **Warning**: This configuration is unsecure because it uses self-signed certificates.
```
Comment (Neudrino) [inline docs/readmes/orc8r/deploy_using_juju.md:56]: I did not know, that the syntax
```markdown
> **Warning**
```
will cause a special rendering in Github. This is magic. :magic_wand: 

FYI Also see https://github.com/community/community/discussions/16925. :wink:

**Review by Neudrino — APPROVED** [review]

**Review by saltiyazan — APPROVED** [review]

**Comment by gruyaume** [comment]
@Neudrino Can you re-run the `rebase` check? I just re-ran it locally and it passed. (I can't make jobs run manually because of my permissions)

**Comment by Neudrino** [comment]
> @Neudrino Can you re-run the `rebase` check? I just re-ran it locally and it passed. (I can't make jobs run manually because of my permissions)

Hi @gruyaume ,
`rebase` check is not a required check, so for this special docs PR, it is fine as is.
It would not turn green anyway, if I rerun, as the state of your git history does not change by this. You would need to rebase on current master and push again, to get it green.

**Comment by gruyaume** [comment]
@Neudrino Anything I can do to get this merged?

**Comment by Neudrino** [comment]
The `ready2merge` label should be applied, such that people know, that you see it fit for merging. Also [see the documentation](https://github.com/magma/magma/wiki/Contributing-Code#code-review-process) please.

I will merge this now anyway.

**Pull request merged**